manifest.json

```diff
@@ -5,10 +5,5 @@
   "documentation": "",
   "language": "node",
   "terraform": false,
-  "monorepo": false,
-  "dynamo_tables": [
-    {
-      "test-results": "test-results.json"
-    }
-  ]
-}
+  "monorepo": false
+}
```

Comment:
Miniscule change but can you run the `format` script on this please, it seems to want an empty line at the end of the manifest json that was updated.

Reply:
done :)